components/property_info.rst

```diff
@@ -411,6 +411,37 @@ library is present::
     $phpDocExtractor->getShortDescription($class, $property);
     $phpDocExtractor->getLongDescription($class, $property);
 
+PhpStanExtractor
+~~~~~~~~~~~~~~~
+
+.. note::
+
+    This extractor depends on the `phpstan/phpdoc-parser`_ and
+    `phpdocumentor/reflection-docblock`_ libraries.
+
+This extractor fetches informations thanks to the PHPStan parser. This
+extractor gather information from annotations of the comment section
+of properties and methods, such as ``@var``, ``@param`` or ``@return``::
+
+    // Foo.php
+    class Foo
+    {
+        private string $bar;
+
+        /**
+         * @param string $bar
+         */
+        public function __construct(string $bar = 'baz') {
```

Comment:
I don't see any value in annotating a typed parameter and property.
Maybe should you remove native types in order to let `@param` do the job?

Reply:
Definitely!